tool/webqa-process-schemaorg.js

```diff
@@ -947,12 +883,21 @@ class SchemaProcessor {
                 })
             ];
             recursiveAddStringValues(args[0], this._prefix + typename + '_name');
+            if (typename !== 'Thing') {
+                // override name so we can apply a custom string_values annotation
+                const arg = new Ast.ArgumentDef(null, Ast.ArgDirection.OUT, 'name', Type.String, {
+                    nl: {},
+                    impl: {
+                        'org_schema_type': new Ast.Value.String('Text'),
+                        'filterable': new Ast.Value.Boolean(false) // no filter on name, if it has ner support, we'll generate prim for it
+                    }
+                });
+                recursiveAddStringValues(arg, this._prefix + typename + '_name');
+                args.push(arg);
```

Comment:
This is wrong and just sneaked in right? We don't want a `name` property.

Reply:
We still have the name in process step to allow us understand which table has name, and which id has ner support, we then remove name at the end of trimming

Reply:
this is actually what the bug was

Reply:
Oh nice catch, but ugh this needs to be written down